### PR TITLE
zh-CN: Translations of some simplified Chinese

### PR DESCRIPTION
**Pull request by ghost**
For some groups in 'working-groups.md', there're some untranslated English phrases, we should tranlate them into Chinese.

### CODE REVIEW

**Comment by ghost** [comment]
@nodejs/nodejs-cn , please have a review, thanks!

File: locale/zh-cn/about/working-groups.md [hunk 1]
```diff
@@ -10,19 +10,19 @@ title: 工作组
...
-* [Evangelism](#evangelism)
-* [i18n](#i18n)
-* [Release](#release)
-* [Security](#security)
-* [Streams](#streams)
-* [Website](#website)
-
-### [Addon API](https://github.com/nodejs/nan)
+* [API 插件组](#addon-api)
+* [标杆分析法组](#benchmarking)
```
Comment (willin) [inline locale/zh-cn/about/working-groups.md]: 看习惯Benchmark了，这么翻很变扭。

`基准测试`

File: locale/zh-cn/about/working-groups.md [hunk 2]
```diff
@@ -10,19 +10,19 @@ title: 工作组
...
+* [编译组](#build)
```
Comment (willin) [inline locale/zh-cn/about/working-groups.md]: `构建`，编译是 `compile`

File: locale/zh-cn/about/working-groups.md [hunk 3]
```diff
@@ -10,19 +10,19 @@ title: 工作组
...
+* [诊断组](#diagnostics)
+* [Docker组](#docker)
+* [宣传组](#evangelism)
+* [国际化翻译组](#i18n)
+* [发布组](#release)
+* [安全组](#security)
+* [‘流’工作组](#streams)
```
Comment (willin) [inline locale/zh-cn/about/working-groups.md]: 也看习惯 Stream 了， 流为什么要加引号？

Comment (ghost) [reply locale/zh-cn/about/working-groups.md]: 不加引号感觉凸显不出效果，加了突出 Stream 的含义。要不我这里还是保留Stream吧。

**Review by willin — APPROVED** [review]
LGTM

**Pull request merged**